### PR TITLE
add Footer to work routes, make mobile listing page footer disappear

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
Move Footer to pages/work so it renders automatically

## How to Test
- [ ] Visit /work/home, /work/account, /work/trips, /work/search and see Footer
- [ ] Visit /work/listings/:id and see Footer on desktop but no Footer on tablet/mobile

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/pages/listing/Listing.tsx [hunk 1]
```diff
@@ -20,7 +18,7 @@ const Listing = ({ match }: RouterProps) => (
         <ListingPage {...data.listing} />
       }
     </Query>
-    <Footer />
+    <DetailedFooter className="d-none d-lg-block" />
```
Comment (kcvan) [inline src/pages/listing/Listing.tsx]: actually now that `<DetailedFooter>` takes a classname, i think you can just export the default `<Footer>` and use react router to handle the all the footers again

File: src/components/work/Footer/Footer.tsx [hunk 2]
```diff
@@ -69,9 +69,10 @@ const socialData = [
   },
 ];
 
-const Footer = () => (
+export const Footer = () => (
   <Switch>
     <Route path="/work/account" component={DetailedFooter} />
+    <Route exact path="/work/listings/:id" component={NoopComponent} />
```
Comment (kcvan) [inline src/components/work/Footer/Footer.tsx]: we can change `component=` to `render=` and pass down className as a prop for the component

File: src/pages/listing/Listing.tsx (hunk 1 above)
Comment (jerminatorhits) [reply src/pages/listing/Listing.tsx]: 🙏

File: src/components/work/Footer/Footer.tsx (hunk 2 above)
Comment (jerminatorhits) [reply src/components/work/Footer/Footer.tsx]: ty

**Comment by kcvan** [comment]
LGTM!

**Comment by tc** [comment]
@jerminatorhits  conflict

**Pull request merged**